csharp/ql/src/csharp.qll

```diff
@@ -34,11 +34,5 @@ import semmle.code.csharp.dataflow.DataFlow
 import semmle.code.csharp.dataflow.TaintTracking
 import semmle.code.csharp.dataflow.SSA
 
-/** DEPRECATED: Use `ControlFlow` instead. */
-deprecated module ControlFlowGraph {
```

Comment:
I guess we have to wait removing this one, as no deprecation warnings are currently given for deprecated modules.